docker/multi-process/README.md

```diff
@@ -0,0 +1,103 @@
+Huginn for docker with multiple container linkage
+=================================================
+
+This image runs a linkable [Huginn](https://github.com/cantino/huginn) instance.
+
+There is an automated build repository on docker hub for [cantino/huginn](https://registry.hub.docker.com/builds/github/cantino/huginn/).
```

Comment:
No, good catch looks like they changed their URL structure, I'll update it in a second.